src/USER-MEAMC/meam.h

```diff
@@ -18,6 +18,7 @@
 
 #include <cmath>
 #include <cstring>
```

Comment:
IWYU: with str_to_lat updated, there is nothing using functions from cstring left in the header.